libs/langgraph/langgraph/pregel/loop.py

```diff
@@ -263,8 +264,28 @@ def put_writes(self, task_id: str, writes: Sequence[tuple[str, Any]]) -> None:
         """Put writes for a task, to be read by the next tick."""
         if not writes:
             return
+        # deduplicate writes to special channels, last write wins
```

Comment:
ooc: why do we know we don't have to deduplicate if one of the writes is to a regular channel?

Reply:
Regular channels can't be written to more than once (writes after the first are ignored)